playbooks/files/plugins/elasticsearch.py

```diff
@@ -136,5 +139,6 @@ def main():
 
 
 if __name__ == '__main__':
-    with print_output():
+    args = options = parse_args()
```

Comment:
Yes this was left because the old object was "option". So to keep things working I left it as-is.